### DISCUSSION TITLE
Display a message when there is no internet connection (local or wifi)

### DISCUSSION THREAD

**Discussion by aurelienb1** [SafeExamBrowser/seb-win-refactoring#327] [Ideas]
Hi,

It would be interesting if SEB displayed a message when there is a loss of internet connection (local or wifi).

We have had several cases where the answers never registered because of disconnection and the user could not see it.

Regards

**Comment by dbuechel** [comment]
Thanks for your input. We actually do have this already tracked in our internal backlog, currently planned to be released with version 3.4.0.
